cachito/workers/tasks/general.py

```diff
@@ -18,10 +18,48 @@
     "failed_request_callback",
     "fetch_app_source",
     "set_request_state",
+    "get_request_state",
+    "check_state",
 ]
 log = logging.getLogger(__name__)
 
 
+def get_request_state(request_id):
+    """
+    Get the state of the request.
+
+    :param int request_id: the Cachito request ID this is for
+    """
+    from cachito.workers.requests import requests_auth_session
+
+    config = get_worker_config()
+    request_url = f'{config.cachito_api_url.rstrip("/")}/requests/{request_id}'
```

Comment:
There is `_get_request_url` doing the same thing. It would be good to make it as public and reuse it to avoid duplicate.